main/main.cpp

```diff
@@ -1351,6 +1351,11 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 
 	ResourceUID::get_singleton()->load_from_cache(); // load UUIDs from cache.
 
+	if (ProjectSettings::get_singleton()->has_custom_feature("dedicated_server")) {
+		audio_driver = "Dummy";
+		display_driver = "headless";
```

Comment:
Would be great is these weren't magic strings...  (Not sure if there is a global somewhere, but would be a great refactor to add one if there isn't)/

Reply:
These same magic strings are hardcoded for the "--headless" CLI option:

https://github.com/godotengine/godot/pull/69546/files/d0b991ad029e7f038ce53665067c8ea7637000ea#diff-18b7c19417a461e4793bb2157831659d96ede991391e10ee5bc51f19453b7685L978

We could maybe have some constants for these at least?

Reply:
Ya, that's what I suspected, which is why I was thinking thinking it would would be a good refactor & consolidate these into a common shared constant.